### ISSUE TITLE
Agent doesn't move during training

### ISSUE THREAD

**Issue by dr1nfinite** [Unity-Technologies/ml-agents#1762]
I finished the tutorial for: https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Learning-Environment-Create-New.md

But while training, AgentAction is receiving nothing. I added the line:
    Debug.Log("V0: " + vectorAction[0] + " V1:" + vectorAction[1] + " ControlSignal: " + controlSignal + " Speed: " + speed);

Which outputs: 
    V0: 0 V1:0 ControlSignal: (0.0, 0.0, 0.0) Speed: 10

Once per frame, hundreds of times per second. Python throws no errors, nor does Unity. Here's that output (which stay static for hours):

    INFO:mlagents.trainers:{'--curriculum': 'None',
     '--docker-target-name': 'None',
     '--env': 'None',
     '--help': False,
     '--keep-checkpoints': '5',
     '--lesson': '0',
     '--load': False,
     '--no-graphics': False,
     '--num-runs': '1',
     '--run-id': 'StartyAIBuild-1',
     '--save-freq': '50000',
     '--seed': '-1',
     '--slow': False,
     '--train': True,
     '--worker-id': '0',
     '<trainer-config-path>': 'config/trainer_config.yaml'}
    INFO:mlagents.envs:Start training by pressing the Play button in the Unity Editor.
    INFO:mlagents.envs:
    'Academy' started successfully!
    Unity Academy name: Academy
        Number of Brains: 1
        Number of Training Brains : 1
        Reset Parameters :

    Unity brain name: RollerBallBrain
        Number of Visual Observations (per agent): 0
        Vector Observation space size (per agent): 8
        Number of stacked Vector Observation: 1
        Vector Action space type: continuous
        Vector Action space size (per agent): [2]
        Vector Action descriptions: ,
    2019-02-25 19:05:00.163118: I T:\src\github\tensorflow\tensorflow\core\platform\cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was
     not compiled to use: AVX2
        INFO:mlagents.envs:Hyperparameters for the PPO Trainer of brain RollerBallBrain:
        batch_size:     10
        beta:   0.005
        buffer_size:    100
        epsilon:        0.2
        gamma:  0.99
        hidden_units:   128
        lambd:  0.95
        learning_rate:  0.0003
        max_steps:      5.0e2
        normalize:      False
        num_epoch:      100
        num_layers:     2
        time_horizon:   64
        sequence_length:        64
        summary_freq:   50
        use_recurrent:  False
        summary_path:   ./summaries/StartyAIBuild-1-0_RollerBallBrain
        memory_size:    256
        use_curiosity:  False
        curiosity_strength:     0.01
        curiosity_enc_size:     128
        model_path:     ./models/StartyAIBuild-1-0/RollerBallBrain
